src/postgres/src/pl/plpgsql/src/pl_gram.y

```diff
@@ -798,19 +798,16 @@ decl_collate	:
 					{ $$ = InvalidOid; }
 				| K_COLLATE T_WORD
 					{
-						ybc_not_support(@1, "COLLATE", 1127);
```

Comment:
These do not appear to be the correct `COLLATE` cases required by the `IMPORT FOREIGN SCHEMA`.